documentation/docs/examples/how-to-run-examples.mdx

```diff
@@ -42,8 +42,8 @@ the following `.env` file:
 ```bash
 MONGO_URL=mongodb://admin:admin@localhost:27017
 DB_NAME=integration-service-db
```

Comment:
sometimes we have `integration-service-db` and sometimes `integration-service` can you replace all with integration-service please

Reply:
Updated